dbl/README.md

```diff
@@ -26,7 +26,7 @@ Unless you explicitly state otherwise, any contribution intentionally submitted
 
 [//]: # (badges)
 
-[crate-image]: https://img.shields.io/crates/v/dbl.svg
+[crate-image]: https://img.shields.io/crates/v/dbl.svg?logo=rust
```

Comment:
BTW I am not sure about this change Rust log in a badge is bit too noisy for my personal taste and we probably should either do it for all crates or none.

Reply:
I like it, it's still recognizable as the Rust logo and I've generally been adding it to new crates I release. But if you oppose it I can remove it.

Reply:
It's better to remove it from this PR. If we are to change badges, It's better to do it in a dedicated PR for all crates in the repo.